contracts/libraries/RemainingInvalidatorLib.sol

```diff
@@ -75,6 +75,6 @@ library RemainingInvalidatorLib {
      * @return result The remaining invalidator for a fully filled order.
      */
     function fullyFilled() internal pure returns(RemainingInvalidator) {
-        return RemainingInvalidator.wrap(1);
+        return RemainingInvalidator.wrap(type(uint256).max);
```

Comment:
It seems like the documentation for `RemainingInvalidatorLib` library needs to be updated.

> When the order is filled, the remaining invalidator is 1

Currently, it is `0xffffffffffffffffffffffffffffffffffffffffffffffffffffffffffffffff`

Reply:
fixed